inc/customizer/extend-custom-controls/src/color-group/color-group-component.js

```diff
@@ -10,13 +15,96 @@ const ColorGroupComponent = props => {
 		name
 	} = props.control.params;
 
+	const linkedSubColors = AstraBuilderCustomizerData.js_configs.sub_controls[name];
+	const colorGroup = [],
+		colorGroupDefaults = [],
+		tooltips = [];
+
+	Object.entries( linkedSubColors ).map( ( [ key,value ] ) => {
+		colorGroup[value.name] = wp.customize.control( value.name ).setting.get();
+		colorGroupDefaults[value.name] = value.default;
+		tooltips[value.name] = value.title;
+	});
+
+	const[ state , setState ] = useState(colorGroup);
+
+	const handleChangeComplete = ( key, color='' ) => {
+		let updateState = {
+			...state
+		};
+
+		let value;
+
+		if (typeof color === 'string') {
+			value = color;
+		} else if (undefined !== color.rgb && undefined !== color.rgb.a && 1 !== color.rgb.a) {
+			value = `rgba(${color.rgb.r},${color.rgb.g},${color.rgb.b},${color.rgb.a})`;
+		} else {
+			value = color.hex;
+		}
+		
+		updateState[key] = value;
+		wp.customize.control( key ).setting.set(value);
+		
+		setState(updateState);
+	};
+
 	if (label) {
 		htmlLabel = <span className="customize-control-title">{label}</span>;
 	}
 
 	if (help) {
 		htmlHelp = <span className="ast-description">{help}</span>;
-	}
+	}	
+
+	let optionsHtml = Object.entries( state ).map( ( [ key,value ] ) => {
+		let tooltip = tooltips[key] || __('Color', 'astra');
+		let html = (
```

Comment:
```suggestion
		return (
```

Since this is a one-off variable, this can be removed and returned right away, right?

Reply:
If this is done you will have remove return from below

Reply:
Done